### DISCUSSION TITLE
Sigma Rule via Hash Codes

### DISCUSSION THREAD

**Discussion by ekky19** [Security-Onion-Solutions/securityonion#7614] [Unsupported Versions] [answered]
Hello All,

Is there any website that shares all the well-known malware's hash codes so that we can create a single sigma rule to scan the network by using those hash codes?

**Comment by defensivedepth** [comment, marked as answer]
Assuming you are looking for file hashes?  You would need to find some type of "threat intel" feed. Just be aware of a couple things:

- Hashes are very low on the `Pyramid of Pain` - they certainly have value, but it's limited, as David Bianco outlines in the post:
http://detect-respond.blogspot.com/2013/03/the-pyramid-of-pain.html

- Adding a large number of hashes to a sigma rule could cause performance issues for your Elasticsearch backend - make sure to test before deploying something like that to production.
